variable expansion of CCARGS ends up in a mess for this check

since the build would fail later anyway if C23 bool was there, we don't care
about the early check
diff -wpruN --no-dereference '--exclude=*.orig' a~/makedefs a/makedefs
--- a~/makedefs	1970-01-01 00:00:00
+++ a/makedefs	1970-01-01 00:00:00
@@ -1234,25 +1234,6 @@ ${WARN='-Wall -Wno-comment -Wformat -Wim
 	-Wparentheses -Wstrict-prototypes -Wswitch -Wuninitialized \
 	-Wunused -Wno-missing-braces -fno-common'}
 
-# C23 introduces a built-in type 'bool' (size 1) which breaks past Postfix
-# practice where 'bool' is an alias for 'int' (typically, size 4).
-# Fortunately, -std=gnu17 works with both gcc and clang.
-cat >makedefs.test.c <<'EOF'
-/* This appears in mail_params.h */
-typedef int bool;
-EOF
-
-if eval ${CC} ${CCARGS} -c makedefs.test.c >/dev/null 2>&1
-then
-    : No further action needed.
-elif eval ${CC} -std=gnu17 ${CCARGS} -c makedefs.test.c >/dev/null 2>&1
-then
-    CCARGS="-std=gnu17 $CCARGS"
-else
-    error 'Please specify "CC=..." or "CCARGS=..." option to disable C23 bool'
-fi
-rm -f makedefs.test.[co]
-
 # Extract map type names from -DHAS_XXX compiler options.  We avoid
 # problems with tr(1) range syntax by using enumerations instead,
 # and we don't try to figure out which awk versions have tolower().
